### ISSUE TITLE
Reflect actual project status

### ISSUE THREAD

**Issue by johnjohndoe** [okfde/2013.archiv.codefor.de#159]
Afaik all projects are tagged with `In progress` ... which is **misleading** for visitors or even members of the labs. Finished projects such as the [`Tempelhofer Feld`](http://codefor.de/projekte/2014-05-19-be-tempelhof.html) should be marked as `finished` and projects [without active members](http://codefor.de/projekte/2014-06-26-altglas.html) should tell about what they are **searching for** such as developers, ...

Another option would be to display the **latest activity** of the project on GitHub such as: `latest commit: 2 days ago` or similar.

**Comment by elf-pavlik** [comment]
:+1: maybe also worth trying to take advantage of http://hackdash.org/
ping @danzajdband

**Comment by streiten** [comment]
'In Progress' is default. 
You can set the project status through a status attribute in front-matter (e.g. 'status: finished')

**Comment by johnjohndoe** [comment]
Yes. I found two projects which actually changed the status of their project. All other use `In progress`. I would love to see something more informative as mentioned previously. Please share your thoughts.

**Comment by elf-pavlik** [comment]
@johnjohndoe do you come to the lab this evening? we could discuss it IRL

**Comment by johnjohndoe** [comment]
I just saw that the `status` of a project only changes in the projects overview page:

**From the projects overview page:**
![tempelhof-overview](https://cloud.githubusercontent.com/assets/144518/5619401/b9ca6d58-9521-11e4-84bf-a18df879ed83.png)

... but it does not render the same `status` on the detail page.

**From the project detail page:**
![tempelhof-details](https://cloud.githubusercontent.com/assets/144518/5619405/c0e05b48-9521-11e4-8063-462ee08a8ead.png)

Is this a bug?

**Comment by elf-pavlik** [comment]
hmmm... looks like something goes wrong here:
- https://github.com/okfde/codefor.de/blob/gh-pages/_includes/project-teaser-slim.html#L48

those lines look working fine:
- https://github.com/okfde/codefor.de/blob/gh-pages/_includes/project-teaser.html#L28
- https://github.com/okfde/codefor.de/blob/gh-pages/_includes/project-teaser.html#L42

**Comment by streiten** [comment]
uh oh.. sorry for that. fixed now!

**Comment by johnjohndoe** [comment]
[Much better](https://github.com/okfde/codefor.de/commit/9d5c8e7b5fd149d2b7285533ffab5624458f82c9), thanks @RU-Alex

**Comment by jfilter** [comment]
I guess it was fixed and we close this issue?

**Comment by johnjohndoe** [comment]
@jfilter, as far as I can tell the initial issue that projects do **not** reflect their actual status on their project page still remains **unresolved**. :-1: 
The minor issue with regards to the inconsistent detail page status was resolved. Please note that I fixed the link to the relevant commit.

**Comment by jfilter** [comment]
Just to clarify: The software is working fine now. But we have to go over each project and check whether the current status is correct, right? I agree that too many projects are not marked as finished/discontinued.

**Comment by johnjohndoe** [comment]
Yes, that is my understanding of the technical and project states.
